v-model and `null` value on radio button with vue-tsc v1.1.3

The following component throws with v1.1.13:

```vue
<script setup lang="ts">
import { ref } from 'vue'
const count = ref<number | null>(null)
</script>

<template>
  <input v-model="count" type="radio" v-for="i in 3" :key="i" :value="i" />
</template>
```

with:
```
src/App.vue:7:10 - error TS2322: Type 'number | null' is not assignable to type 'any[] | Set<any> | Booleanish | undefined'.
  Type 'null' is not assignable to type 'any[] | Set<any> | Booleanish | undefined'.

7   <input v-model="count" type="radio" v-for="i in 3" :key="i" :value="i" />
           ~~~~~~~

  node_modules/@vue/runtime-dom/dist/runtime-dom.d.ts:646:3
    646   checked?: Booleanish | any[] | Set<any> // for IDE v-model multi-checkbox support
          ~~~~~~~
    The expected type comes from property 'checked' which is declared here on type 'ElementAttrs<InputHTMLAttributes>'
```

`undefined` is allowed, but `null` is no longer possible

--- a/packages/vue-language-core/schemas/vue-tsconfig.schema.json
+++ b/packages/vue-language-core/schemas/vue-tsconfig.schema.json
@@ -109,12 +109,6 @@
 						"": {
 							"input": true
 						},
-						"checked": {
-							"input": [
-								{ "type": "checkbox" },
-								{ "type": "radio" }
-							]
-						},
 						"value": {
 							"input": { "type": "text" },
 							"textarea": true,

--- a/packages/vue-language-core/src/utils/ts.ts
+++ b/packages/vue-language-core/src/utils/ts.ts
@@ -169,12 +169,6 @@ export function resolveVueCompilerOptions(vueOptions: Partial<VueCompilerOptions
 			'': {
 				input: true
 			},
-			checked: {
-				input: [
-					{ type: 'checkbox' },
-					{ type: 'radio' }
-				]
-			},
 			value: {
 				input: { type: 'text' },
 				textarea: true,

